sdk/appcenter/src/main/java/com/microsoft/appcenter/UserInformation.java

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-package com.microsoft.appcenter.identity;
+package com.microsoft.appcenter;
```

Comment:
Maybe I didn't explain correctly in previous comments.
Just revert this file, it needs to stay `package com.microsoft.appcenter.identity;` like before.

Reply:
But we can't import the package com.microsoft.appcenter.identity in the appcenter core package, because identity package is dependent on it.

Reply:
Oh it's in core now ok. Then root of the core is fine.